playbooks/roles/kibana/templates/nginx.conf.j2

```diff
@@ -0,0 +1,47 @@
+upstream elasticsearch_server {
+   server 127.0.0.1:9200;
+}
+
+server {
+  # Kibana server, templated by ansible
+
+  {% if NGINX_ENABLE_SSL %}
+
+  listen {{ KIBANA_SERVER_NAME }}:{{ KIBANA_NGINX_PORT }};
+  listen {{ KIBANA_SERVER_NAME }}:{{ KIBANA_SSL_NGINX_PORT }};
+
+  ssl_certificate /etc/ssl/certs/{{ NGINX_SSL_CERTIFICATE|basename }};
+  ssl_certificate_key /etc/ssl/private/{{ NGINX_SSL_KEY|basename }};
+
+  {% else %}
+  listen {{ KIBANA_SERVER_NAME }}:{{ KIBANA_NGINX_PORT }};
```

Comment:
From the looks of this, we always include this line, even when HTTPS, so I think we can remove this from the `if` block like:

```jinja2
listen {{ KIBANA_SERVER_NAME }}:{{ KIBANA_NGINX_PORT }};
{% if NGINX_ENABLE_SSL %}
listen {{ KIBANA_SERVER_NAME }}:{{ KIBANA_SSL_NGINX_PORT }};
ssl_certificate /etc/ssl/certs/{{ NGINX_SSL_CERTIFICATE|basename }};
ssl_certificate_key /etc/ssl/private/{{ NGINX_SSL_KEY|basename }};
{% endif %}
```